# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc
# Commit: b94935661622
# Full Hash: b949356616223660996acf39e1e353e7bbd2a60d
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 4
# Description:
#   Bug 1376873 - Use Call interface in AudioConduit; r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D7441
# ==============================================================================

diff -r 6eca45c83131 -r b94935661622 media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc
--- a/media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc	Mon Mar 26 16:19:23 2018 -0400
+++ b/media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc	Tue Aug 21 13:39:53 2018 -0400
@@ -87,7 +87,7 @@
 }
 
 void ChannelProxy::SetReceiveCsrcAudioLevelIndicationStatus(bool enable, int id) {
-  //RTC_DCHECK(worker_thread_checker_.CalledOnValidThread());
+  RTC_DCHECK(worker_thread_checker_.CalledOnValidThread());
   int error = channel()->SetReceiveCsrcAudioLevelIndicationStatus(enable, id);
   RTC_DCHECK_EQ(0, error);
 }
